includes/entry.inc.php

```diff
@@ -70,6 +71,35 @@
 				$rstatus = save_read_status($connid, $user_id, $id);
 			}
 			
+			if (isset($_SESSION[$settings['session_prefix'] . 'user_id']) && intval($entrydata['locked'])==0) {
+				// vote handling
+				$user_id = $_SESSION[$settings['session_prefix'] . 'user_id'];
+        $result = @mysqli_query($connid, "SELECT TRUE AS 'exists' FROM " . $db_settings['userdata_table'] . " WHERE user_id = " . intval($user_id) . " AND voting_allowed >= 1 LIMIT 1") or raise_error('database_error', mysqli_error($connid));
```

Comment:
Is `voting_allowed` a true/false (0/1) switch or an integer that can be greater than 1? Same for posting.inc.php line 326 ff.